util/check_translation.py

```diff
@@ -221,6 +221,8 @@ def translate_timeline(line, trans):
     for old, new in trans['replaceSync'].items():
         did_work = did_work or re.search(old, text)
         text = re.sub(old, new, text)
+    # escape \d in timeline *.txt files to avoid KeyError/ re.error: bad escape \d
```

Comment:
Should more things be escaped here?

Reply:
this is currently the only issue i had (nearly done with stormblood)

Reply:
Which file was this failing on?

Reply:
it was on o10s.js
the command i used was: `python check_translation.py -f o10s.js -t de -gm all`

Reply:
Thanks.  Sorry, I haven't had a chance to look into this more yet.  I wanted to understand the problem better and see if there was a more general solution.